docs/installation/index.md

```diff
@@ -18,19 +18,36 @@ The Autoware Foundation provides no support on other platforms than those listed
 
 Refer to [REP-2000](https://www.ros.org/reps/rep-2000.html) for the system dependencies.
 
-## Installation steps of Autoware
+## Installation of Autoware
 
-See [here](autoware) for how to install Autoware.
+There are two ways to set up Autoware. Choose one according to your preference.
 
-## Installation steps of tools for users
+### 1. Docker installation
+
+Docker can ensure that all developers in a project have a common, consistent development environment.
+It is recommended for beginners, light users, people who do not use Ubuntu.
+
+See [here](https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/installation/autoware/docker-installation.md) for the detailed steps.  
+Refer to [autowarefoundation/autoware/docker/README.md](https://github.com/autowarefoundation/autoware/blob/main/docker/README.md) as well.
```

Comment:
@kenji-miyake On reflection, I feel that the Docker README file with example run commands should be added to the Autoware documentation. What do you think about this idea?

Reply:
@LalithVipulananthan Hmm? I guess it's here. Could you explain it a bit more in detail?
https://github.com/autowarefoundation/autoware-documentation/blob/9834c0e3d525648f3bcef1d22c8649b7e3386eb9/docs/installation/autoware/docker-installation.md?plain=1#L60

Reply:
I understand the Docker installation page has a link to the advanced usage commands that are in the README of autowarefoundation/autoware/docker, but I just felt that for consistency these commands should be part of the core documentation. It's not that big a deal so if you think that information is fine where it is then that's okay! 😄 

That said, if the Docker installation page contains a reference to the README.md file, then we don't need to have a link to it on this page as well right?

Reply:
> but I just felt that for consistency these commands should be part of the core documentation.

I think details should be in `autoware/docker/README.md` for easy maintenance and that the summary should be in autoware-documentation.

> That said, if the Docker installation page contains a reference to the README.md file, then we don't need to have a link to it on this page as well right?

Right, we can remove the duplicated link from this page.

Reply:
I think either would be fine, but it might be easier to understand as it is now. 
autoware-documentation installation page should include step by step installation, and detailed information don't have to be included there. 
Also, people who just take a glance at docker readme in github https://github.com/autowarefoundation/autoware/tree/main/docker might need some information.